VAMobile/ios/RNAuthSession.swift

```diff
@@ -76,6 +77,21 @@ class RNAuthSession: NSObject, RCTBridgeModule, ASWebAuthenticationPresentationC
     session?.prefersEphemeralWebBrowserSession = true
     session?.start()
   }
+ 
+ /// Clears the devices cookies
+ /// - Parameters:
+ ///   - useWebKit: bool to use webkit or not
+ ///   - resolve: React Native Promise resolver.
+ /// - Returns: resolves true or false if it was able to clear the users cookies
+ @objc(resolver:)
+ func clearCookies(resolver resolve: @escaping RCTPromiseResolveBlock)-> Void {
```

Comment:
i disagree. since this has to do with auth and login I feel they should be in the same file. Since this is just being called in the authSlice the native module also called in there is RNAuth for iOS and CustomTabs in Android so I feel it is appropriately placed